src/modules/ekf2/EKF/ekf.cpp

```diff
@@ -346,9 +346,15 @@ bool Ekf::resetGlobalPosToExternalObservation(const double latitude, const doubl
 
 		} else {
 			ECL_INFO("fuse external observation as position measurement");
+
+#if defined(CONFIG_EKF2_WIND)
+			_enforce_wind_fusion = true;
```

Comment:
Doing this before and after the call is pretty hacky to be honest. Can you at least add a comment?